### PR TITLE
Add Terasky Ltd to ADOPTERS list

### PR DESCRIPTION
**Pull request by davidmirror-ops**
Signed-off-by: David Espejo <daespejo@vmware.com>

**What this PR does / why we need it**:
Add TeraSky Ltd. to the list of ADOPTERS organizations

**Which issue(s) this PR fixes**
- Fixes #
N/A
**Special notes for your reviewer**:

**Release note**:
```
release-note
```

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
 # Octant Adopters
+<a href="https://www.terasky.com" border="0" target="_blank"><img alt="terasky.com" src="site/static/img/adopters/TeraSky-Color.png" height="50"></a>&nbsp; &nbsp; &nbsp;
```
Comment (GuessWhoSamFoo) [inline ADOPTERS.md:2]: src path doesn't match path below?

Comment (davidmirror-ops) [reply ADOPTERS.md]: you're right, my bad. Now fixed

**Comment by GuessWhoSamFoo** [comment]
This needs a rebase to pass CI

**Comment by GuessWhoSamFoo** [comment]
Nevermind - the test is broken on master

**Review by GuessWhoSamFoo — APPROVED** [review]

**Pull request merged**